lib/index.js

```diff
@@ -327,14 +331,20 @@ api.getPublicKeys = function(id, actor, options, callback) {
 // FIXME: sync private key label to public key label
 api.updatePublicKey = function(actor, publicKey, callback) {
   async.auto({
-    checkPermission: function(callback) {
+    clean: function(callback) {
+      // Removing owner forces the get function to be called in checkPermission,
+      // so the owner checked is the one in the db, not the key passed in.
+      delete publicKey.owner;
```

Comment:
I have updated the comment section of this API to reflect that the publicKey owner should be sanitized or deleted before being sent to this API.